docs/pages/reference/machine-id/diagnostics-service.mdx

```diff
@@ -132,3 +117,67 @@ performance issues.
 
 They will only be enabled if the `-d`/`--debug` flag is provided when starting
 `tbot`. This is known as **debug mode**.
+
+## Prometheus Metrics
+
+The `tbot` process exposes a number of Prometheus metrics via the `/metrics`
+endpoint of the diagnostics service.
+
+In addition to exporting the standard Go runtime metrics, `tbot` also exports
+custom metrics that reflect the health and performance of the various
+configurable services.
+
+## Advice
+
+When monitoring the health of `tbot`, there are largely three categories of
+metrics you should consider:
```

Comment:
I would make the connection between these categories and the metrics listed later on the page more explicit. Otherwise, I'm not immediately sure whether the metrics we recommend considering are listed.